<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSolanizer</class>
 <widget class="QMainWindow" name="QSolanizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSolanizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="widget">
           <attribute name="title">
            <string>Tag</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCustomPlot" name="wPowerCurve" native="true">
                <zorder>calendarWidget</zorder>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QCalendarWidget" name="calendarWidget"/>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QDateEdit" name="dateEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget">
           <attribute name="title">
            <string>Monat</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_4">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCustomPlot" name="widget" native="true"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QTreeWidget" name="tMonthSelection">
                  <property name="showDropIndicator" stdset="0">
                   <bool>false</bool>
                  </property>
                  <attribute name="headerVisible">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Jahr</string>
                   </property>
                  </column>
                  <item>
                   <property name="text">
                    <string>2013</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>Januar</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Februar</string>
                    </property>
                   </item>
                  </item>
                  <item>
                   <property name="text">
                    <string>2014</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>MÃ¤rz</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>April</string>
                    </property>
                   </item>
                  </item>
                  <item>
                   <property name="text">
                    <string>2015</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>Juni</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Juli</string>
                    </property>
                   </item>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_10">
                  <property name="title">
                   <string>Benutzerdefinierter Plot</string>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Start</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="dateEdit_2"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Ende</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="dateEdit_3"/>
                   </item>
                  </layout>
                  <zorder>label</zorder>
                  <zorder>dateEdit_2</zorder>
                  <zorder>dateEdit_3</zorder>
                  <zorder>label_2</zorder>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Jahr</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1,0">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QGroupBox" name="groupBox_5">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_6">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCustomPlot" name="widget_2" native="true"/>
              </item>
              <item>
               <widget class="QListWidget" name="listWidget">
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>2012</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2013</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2014</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2015</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Gesamt</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1,0">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QGroupBox" name="groupBox_7">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_8">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCustomPlot" name="widget_3" native="true"/>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>GroupBox</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="bTest">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu_Laden"/>
    <addaction name="actionAndere_Anlage_ffnen"/>
    <addaction name="separator"/>
    <addaction name="actionEinstellungen"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="action_ber"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeu_Laden">
   <property name="text">
    <string>neu Laden</string>
   </property>
  </action>
  <action name="actionAndere_Anlage_ffnen">
   <property name="text">
    <string>andere Anlage Ã¶ffnen</string>
   </property>
  </action>
  <action name="actionEinstellungen">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>Ãber</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcp\qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
